admin/bin/dev-shell

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+set -eu
+
+TOPLEVEL=$(git rev-parse --show-toplevel)
+
+#!/bin/bash
+# shellcheck disable=SC2086
+
+set -eu
+
+TOPLEVEL=$(git rev-parse --show-toplevel)
+
+function display_dots() {
```

Comment:
Doesn’t seem necessary

Reply:
The first time around the image takes a few minutes to build, it is nice to see something moving while it happens.